### DISCUSSION TITLE
Fail to trigger automatic re-deploy function after customization

### DISCUSSION THREAD

**Discussion by PanZiwei** [alshedivat/al-folio#552] [Q&A]
Hi all,
I have carefully gone through the README file but I failed to trigger automatic re-deployment after new changes. See my [Git repo](https://github.com/zukailiu/zukailiu.github.io) (I used my friend's repo for test since my GitHub page is occupied already)

Here is what I did:
Follow the [Local setup](https://github.com/alshedivat/al-folio#local-setup) -> Follow the [deployment](https://github.com/alshedivat/al-folio#deployment) part to change the repo only -> Push the local repo and set up the Github Page to the `gh-pages` branch
Everything works well at this moment, I can find the deployment in the Actions part.

However, after I customize the context, I successfully pushed the new changes, finding only the master branch is changed, **nothing happened( no updated commitment) to the `gh-pages` branch, and `pages-build-deployment` is not triggered in the Actions.** As a result, the Github Page is not updated.

I have successfully loaded all the updated websites locally, so I am guessing that it should be something wrong with the "automatically re-deploy your webpage" function?

Would really appreciate it if any advice can be offered! Thanks!

Best,
Ziwei

**Comment by alshedivat** [comment]
how does the [pages section](https://github.com/zukailiu/zukailiu.github.io/settings/pages) of your repo look like?

make sure source is set to `gh-pages` and not to `master`. it seems like the website is trying to use the default github jekyll deployment method instead of using `.github/workflows/deploy.yml`.
